bitbucket/resource_group.go

```diff
@@ -21,6 +24,11 @@ func resourceGroup() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"import_if_exists": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
```

Comment:
This is a bit of a terraform anti-pattern. The normal process is to do an explicit import of a group outside of the plan/apply phase. What is the workflow you're trying to fix with this?

Reply:
We realize the anti-pattern, hence a flag is created with the default value of false. 

Our use case for the import_if_exists flag is for migrations where the groups would already exist in the RDS snapshots that we would restore from. In this case, we can't import the state because we haven't run the terraform apply yet to bring up the bitbucket server.

Reply:
hmm ok that is a bit odd and i'm ok for it to go through as it's a little benign - can you also update the docs here: https://github.com/gavinbunney/terraform-provider-bitbucketserver/blob/master/docusaurus/docs/resource_user.md

Reply:
Updated the resourc_group.md file doc for  add import_if_exist functionality.